jobs/migrations/0004_auto_20180326_1352.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.0.3 on 2018-03-26 13:52
+
```

Comment:
You can squash multiple migrations to one in PRs with https://docs.djangoproject.com/en/1.11/ref/django-admin/#django-admin-squashmigrations

Not that important, but makes the code cleaner.